launcher/minecraft/auth/AccountData.cpp

```diff
@@ -473,6 +527,9 @@ QString AccountData::accountDisplayString() const {
         case AccountType::Mojang: {
             return userName();
         }
+        case AccountType::CustomYggdrasil: {
+            return userName();
+        }
```

Comment:
```suggestion
        case AccountType::Mojang:
        case AccountType::CustomYggdrasil: {
            return userName();
        }
```
the braces do kind of look weird here tbh :woman_shrugging:

Reply:
Done!